applications/serial_lte_modem/src/slm_at_fota.c

```diff
@@ -297,7 +330,30 @@ int handle_at_fota(enum at_cmd_type cmd_type)
 			}
 			if (op == SLM_FOTA_START_APP) {
 				type = DFU_TARGET_IMAGE_TYPE_MCUBOOT;
-			} else {
+			}
+#if defined(CONFIG_SLM_FULL_FOTA)
+			else if (op == SLM_FOTA_FULL_FOTA)  {
+				if (!device_is_ready(flash_dev)) {
+					LOG_ERR("Flash device %s not ready\n", flash_dev->name);
+					return -ENXIO;
+					}
+
+				/* Inside dfu_target_full_modem_cfg there is a check where the
+				 * buffer size has to be multiplication by the block size.
```

Comment:
I do not quite catch the meaning of this comment and debug trace. It would make more sense if the buffer size was configurable via kconfig, but now it is constant. Is there a need for this comment and debug trace from user perspective?

Also, do you mean multiple of block size?